src/components/ConnectBox.js

```diff
@@ -0,0 +1,222 @@
+/**
+ * Connect Box component that renders an information box of
+ * a user (either veteran or parter org) when their corresponding
+ * pin is pressed.
+ *
+ * @prop connection       - either veteran or parter org object
+ *                          (cannot think of a better name?!)
+ *                          requires these properties:
+ *                            - name
+ *                            - roles
+ *                            - image
+ *                            - bio
+ * @prop onClose          - callback to be executed when close button pressed
+ */
+
+import React from 'react';
+import Icon from '@expo/vector-icons/FontAwesome';
+import { StyleSheet, Text, View, Animated, Image } from 'react-native';
+
+import { colors } from '../styles/colors';
+import { layoutStyles, margins } from '../styles/layout';
+import Animations from '../styles/animations';
+import Button from '../components/Button';
+
+export default class ConnectBox extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      animationValue: new Animated.Value(0),
+
+    };
+
+    this.onBoxClose = this.onBoxClose.bind(this);
+  }
+
+  componentDidMount() {
+    Animations.fade(this.state.animationValue, toValue = 1).start();
+  }
+
+  onBoxClose() {
+    Animations.fade(this.state.animationValue).start(() => {
+      this.props.onClose();
+    });
+  }
+
+  getConnectionTitle() {
+    return this.props.connection.roles ?
```

Comment:
the partnering org model doesn't have a function to serialize the roles enum so we may encounter a problem with this when we try to access roles for PO

Reply:
hmm yea I think in that case roles would just be undefined and it would go into the else clause